Gemfile

```diff
@@ -9,6 +9,7 @@ gem 'shared_mustache', '~> 0.1.3'
 gem 'airbrake', '~> 4.0.0'
 gem 'chronic', '~> 0.10.2'
 gem 'govuk_navigation_helpers', '~> 2.0.0'
+gem 'invalid_utf8_rejector', '~> 0.0.1'
```

Comment:
I looked in https://github.com/alphagov/invalid_utf8_rejector/blob/master/lib/invalid_utf8_rejector/version.rb and saw `0.0.1` - I'll update it in the Gemfile.